devops/openshift/sysdig-team.yml

```diff
@@ -1,27 +1,39 @@
-apiVersion: ops.gov.bc.ca/v1alpha1
-kind: SysdigTeam
-metadata:
-  name: 0c27fb-sysdigteam
-  namespace: 0c27fb-tools
-spec:
-  team:
-    description: The Sysdig Team for the OpenShift Project Set SIMS
-    users:
-      - name: nino.samson@gov.bc.ca
-        role: ROLE_TEAM_EDIT
-      - name: bidyashish.kumar@gov.bc.ca
-        role: ROLE_TEAM_EDIT
-      - name: shashank.x.shekhar@gov.bc.ca
-        role: ROLE_TEAM_EDIT
-      - name: jason.tang@gov.bc.ca
-        role: ROLE_TEAM_EDIT
-      - name: gurumoorthy.mohan@aot-technologies.com
-        role: ROLE_TEAM_EDIT
-      - name: dheepak.rr@aot-technologies.com
-        role: ROLE_TEAM_EDIT
-      - name: andrew.signori@aot-technologies.com
-        role: ROLE_TEAM_EDIT
-      - name: andre.pestana@aot-technologies.com
-        role: ROLE_TEAM_STANDARD
-      - name: lewis.chen@aot-technologies.com
-        role: ROLE_TEAM_STANDARD
+apiVersion: template.openshift.io/v1
```

Comment:
did we update the kind to template from the SysdigTeam ?

https://developer.gov.bc.ca/docs/default/component/platform-developer-docs/docs/app-monitoring/sysdig-monitor-setup-team/?utm_source=digital&utm_medium=web&utm_campaign=sysdig-monitor#part-1-compose-the-sysdig-team-object-manifest 

as per the documentation of bcgov devops, its been created, anything changed?

Reply:
The file did not use a template before. I converted it to a template to allow the parameter to be added. The sysdig object should be exactly the same as before: `ops.gov.bc.ca/v1alpha1`. Am I missing something?